terms/demographics/familyHistory.json

```diff
@@ -0,0 +1,36 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema",
+  "$id": "sage.annotations-demographics.familyHistory-0.0.1",
+  "type": "string",
+  "description": "Is there a family history of the disorder/disease",
```

Comment:
I think Unknown could cause some semantic confusion, because there is a big difference between a researcher putting "unknown" because the individuals either didn't respond or weren't asked the question, and "unknown" because the person in question was adopted or for some other reason doesn't know their family history.

Reply:
I agree with Nicole's and Juliane's points.  My preference for these types of keys would be to eliminate the "unknown" option and define the key as Boolean, unless there's a specific use case to do otherwise.  "unknown" doesn't give us any more information than having a missing value in the key would.  People usually leave the column blank instead of filling in "unknown", and this causes some programming headaches for me because without a non-Boolean string value ("unknown") in the data, Python Pandas defaults true/false values internally to Boolean and I then have to take the extra step of converting them to strings in order to validate them using schemas.

Reply:
> this causes some programming headaches for me because without a non-Boolean string value ("unknown") in the data, Python Pandas defaults true/false values internally to Boolean and I then have to take the extra step of converting them to strings in order to validate them using schemas.

I have the opposite problem. I generally want booleans and having extra values is a headache. 😆